providers/k8s/resources/discovery.go

```diff
@@ -822,12 +822,14 @@ func convertImagesToAssets(images map[string]ContainerImage) ([]*inventory.Asset
 
 		ref, err := name.ParseReference(i.resolvedImage, name.WeakValidation)
 		if err != nil {
-			return nil, err
+			log.Error().Err(err).Msg("failed to parse image reference")
```

Comment:
That is a good temporary measure, we need to see how we can still report that correctly.